scp/cc/cpio/client_providers/cloud_initializer/src/azure/no_op_initializer.cc

```diff
@@ -0,0 +1,46 @@
+/*
+ * Portions Copyright (c) Microsoft Corporation
```

Comment:
We copied the gcp implementation, but accidentally added the Microsoft copyright. We made this PR without the fix because testing is expensive with our infrastructure at this moment. We'll fix them within this PR if there is other change that needs to be made before merging.